applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -1034,24 +1030,56 @@ static int cmd_hfclkaudio_drift_comp_disable(const struct shell *shell, size_t a
 	ARG_UNUSED(argc);
 	ARG_UNUSED(argv);
 
-	ctrl_blk.drift_comp.hfclkaudio_comp_enabled = false;
+	ctrl_blk.drift_comp.enabled = false;
```

Comment:
Should set the drift comp state to INIT here (do it the same way as for pres comp)